event-sourcing/event-subscription-registry/src/main/java/uk/gov/justice/subscription/registry/SubscriptionDescriptorDefinitionRegistryProducer.java

```diff
@@ -53,6 +53,10 @@ public SubscriptionDescriptorDefinitionRegistry subscriptionDescriptorRegistry()
             try {
                 final List<URL> subscriptionDescriptorPaths = yamlFileFinder.getSubscriptionDescriptorPaths();
 
+                if (null == subscriptionDescriptorPaths || subscriptionDescriptorPaths.isEmpty()){
```

Comment:
As above...I don't think we need to check for null...as subscriptionDescriptorPaths can not be null